### DISCUSSION TITLE
Show related (nested) revisions in Revisions drawer

### DISCUSSION THREAD

**Discussion by kamil-malinski** [directus/directus#13784] [Feature Requests]
Hi!

I'm using directus as a CMS for articles. My articles-collection stores various data about the article (title, description, intro etc.) and uses a m2a-builder for the article content. With the builder the editor can add various elements like a headline, an image, a paragraph etc. to the article content.

Since those elements are not part of the article-collection but collections on its own the editors can not track the changes they do within the article content. So whenever somebody changes something in a paragraph within an article a new article revision is being created with only the "date updated"-field changed. I understand that the paragraph-collection has its own revisions but there's no easy way to reach the revisions of a certain paragraph from within the article.

What options do I have here? I was thinking to use a hook to concat the texts from the various content element (headline, paragraph etc.) and store it into an article field. This would allow the editor to at least see the changes of the content elements. But it looks like fields that are being updated through a hook don't appear in the list of changed field in the revision system.

Is there any other way to use relational fields with the revision system?

Cheers

Kamil

**Comment by rijkvanzanten** [comment]
Revisions itself do track what parent revision triggered the change, so you could theoretically pull in the revisions (with all nested changes) from the API's /revisions endpoint. That being said, the app currently doesn't utilize this to show related changes included in the revision, so we'll treat this as the feature request to add that information in the app 👍🏻

**Reply by benhaynes** [reply]
This would be an awesome feature to add, nesting related revisions into the parent item. Like you said, we have the data, we just need to fetch it and display it.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
